selfdrive/car/gm/values.py

```diff
@@ -70,13 +83,15 @@ class GMCarInfo(CarInfo):
 
 
 CAR_INFO: Dict[str, Union[GMCarInfo, List[GMCarInfo]]] = {
-  CAR.HOLDEN_ASTRA: GMCarInfo("Holden Astra 2017", harness=Harness.custom),
-  CAR.VOLT: GMCarInfo("Chevrolet Volt 2017-18", footnotes=[Footnote.OBD_II], min_enable_speed=0, harness=Harness.custom),
-  CAR.CADILLAC_ATS: GMCarInfo("Cadillac ATS Premium Performance 2018"),
-  CAR.MALIBU: GMCarInfo("Chevrolet Malibu Premier 2017", harness=Harness.custom),
-  CAR.ACADIA: GMCarInfo("GMC Acadia 2018", video_link="https://www.youtube.com/watch?v=0ZN6DdsBUZo", footnotes=[Footnote.OBD_II]),
-  CAR.BUICK_REGAL: GMCarInfo("Buick Regal Essence 2018"),
-  CAR.ESCALADE_ESV: GMCarInfo("Cadillac Escalade ESV 2016", "ACC + LKAS", footnotes=[Footnote.OBD_II]),
```

Comment:
Not sure if there is value in looking up the explicit package names for all these... I don't know if package names change from year to year either, or how that should be handled here.

Finally, now that we are adding cars without an ASCM, sometimes both camera-based and radar-based are options; both of which would be supportable, but the ASCM variant can do OP long, while the cam cannot (officially). Again - not sure how this would be handled. More of a future concern...